misc.go

```diff
@@ -154,6 +147,27 @@ func postForm(ctx context.Context, client HTTPRequester, endpoint string, values
 	return parseResponseBody(resp.Body, intf, debug)
 }
 
+func postJson(ctx context.Context, client HTTPRequester, endpoint, token string, json []byte, intf interface{}, debug bool) error {
+	reqBody := bytes.NewBuffer(json)
+	req, err := http.NewRequest("POST", SLACK_API+endpoint, reqBody)
```

Comment:
lets keep this method consistent with postForm below and only accept the fully formed URL.